app-localize-behavior.html

```diff
@@ -115,7 +116,9 @@
      * The language used for translation.
      */
     language: {
-      type: String
+      type: String,
+      value: Polymer.defaultLanguage,
```

Comment:
Oof, I don't know about this. We set default values fairly differently -- the element that implements this behaviour can set a `value` for this attribute in its properties block. I don't think this is a good pattern